app/uk/gov/hmrc/eoricommoncomponent/frontend/views/how_can_we_identify_you_utr.scala.html

```diff
@@ -29,6 +29,12 @@
 
 @(utrForm: Form[IdMatchModel], isInReviewMode: Boolean, submit: Call, orgType: EtmpOrganisationType, heading: String = "subscription-journey.how-confirm-identity.utr.heading")(implicit request: Request[_], messages: Messages)
 
+@forHintMessage = @{
+    if(orgType.etmpOrgTypeCode == "0003") messages("subscription-journey.how-confirm-identity.utr.hint")
```

Comment:
Right, this magic strings - without hunting it down it's hard to know what on earth a 0003 means. Lets have a quick catchup